src/ConnectFour/Port/Adapter/Persistence/Repository/PredisOpenGameStore.php

```diff
@@ -7,46 +7,58 @@
 use Gaming\ConnectFour\Application\Game\Query\Model\OpenGames\OpenGame;
 use Gaming\ConnectFour\Application\Game\Query\Model\OpenGames\OpenGames;
 use Gaming\ConnectFour\Application\Game\Query\Model\OpenGames\OpenGameStore;
-use Predis\ClientInterface;
+use Predis\Client;
+use Predis\ClientContextInterface;
 
 final class PredisOpenGameStore implements OpenGameStore
 {
+    private const int GAME_INFO_EXPIRE_TIME = 5;
+
     public function __construct(
-        private readonly ClientInterface $predis,
+        private readonly Client $predis,
```

Comment:
Changed from interface to concrete because `pipeline()` isn't part of the interface definition.